tools/hooks/20-flake8.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+echo "Executing 'flake8' formatter..."
+
+pushd $PROJECT_ROOT > /dev/null
+  python -m flake8        \
```

Comment:
This is not filtered by python file type and the tracked files. Should be similar to black instead of using `./` below.

Reply:
That is excluded in the `.flake8` config. Should I move it here?